src/legacy/status_im/events.cljs

```diff
@@ -108,6 +108,8 @@
   (let [new-account?            (get db :onboarding/new-account?)
         app-in-background-since (get db :app-in-background-since)
         signed-up?              (get-in db [:profile/profile :signed-up?])
+        notifications-settings? (= (get-in db [:view-id])
+                                   :screen/settings.notifications)
```

Comment:
This value is used inside the event that's called when the app returns from being in the background. For example, when we return from outside the app, we'll check if we've returned to the notification settings screen so we can refresh the permissions state.

Reply:
```clojure
(= (get-in db [:view-id])  :screen/settings.notifications)
```
->

```clojure
(= (:view-id db) :screen/settings.notifications)
```

Reply:
I wonder if instead we could add a handler in the screen mounted (by using `use-effect` or something related). If we can't, it's ok to keep it here.